en_partut-ud-test.conllu

```diff
@@ -2467,7 +2467,7 @@
 13	accession	accession	NOUN	S	Number=Sing	11	nmod	_	_
 14	or	or	CCONJ	CC	_	22	cc	_	SpaceAfter=No
 15	,	,	PUNCT	FF	_	22	punct	_	_
-16	where	where	SCONJ	CS	_	17	mark	_	_
+16	where	where	ADV	B	PronType=Rel	17	advmod	_	_
```

Comment:
I guess this is a reduced clause, short for "Where [it is] later". Probably not relative.